sonar-csharp-plugin/pom.xml

```diff
@@ -50,7 +50,6 @@
     <dependency>
       <groupId>org.sonarsource.api.plugin</groupId>
       <artifactId>sonar-plugin-api</artifactId>
-      <version>${sonar.version}</version>
       <scope>provided</scope>
```

Comment:
Same as for comment below: can we have a comment explaining what is providing the sonar-plugin-api (I guess SonarQube/SonarCloud) and why we need the scope defined at this level?